userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1577,29 +1575,23 @@ int32_t scap_bpf_load(
 			}
 		}
 
-		if(online_cpu >= handle->m_dev_set.m_ndevs)
```

Comment:
We don't need this check anymore since the for loop exits when we reach expected number of online cpus.